### DISCUSSION TITLE
Relation "key_value" does not exist

### DISCUSSION THREAD

**Discussion by TheodoreCAshmore** [k3s-io/k3s#7819] [Q&A] [answered]
Hi,

I'm using K3s with  PostgreSQL. I was looking into the database log and it seems that just after creating indexes, a query "SELECT COUNT(*) FROM key_value" is launched, which returns the error 'relation "key_value" does not exist'. It does not seem to cause any further problems, but I was curious as to why that is happening and I didn't seem to find any answer, so I thought of asking here.

**Comment by brandond** [comment, marked as answer]
https://github.com/k3s-io/kine/blob/master/pkg/drivers/generic/generic.go#L140
